src/site/content/en/blog/rebelmouse/index.md

```diff
@@ -0,0 +1,216 @@
+        <td>Hamu és Gyémánt</td>
+        <td>+10.6%</td>
+        <td>+48.2%</td>
+        <td>+13.1%</td>
+        <td>+47.2%</td>
+        <td>+6.9%</td>
+        <td>+35.3%</td>
+      </tr>
+      <tr>
+        <td>Knocksense</td>
+        <td>+37.5%</td>
+        <td>+52.8%</td>
+        <td>+46.9%</td>
+        <td>+64.4%</td>
+        <td>+47.0%</td>
+        <td>+64.6%</td>
+      </tr>
+      <tr>
+        <td>Upworthy</td>
+        <td>+117.6%</td>
+        <td>+50.0%</td>
+        <td>+117.1%</td>
+        <td>+50.8%</td>
+        <td>+114.2%</td>
+        <td>+49.2%</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+But content consumption and time spent per user weren't the only metrics to spike. The number of users visiting and pages accessed also increased among the same categories:
+
+<div class="table-wrapper scrollbar">
+  <table>
+    <thead>
+      <tr>
+        <th aria-hidden></th>
+        <th colspan="2">
+          <strong>Loyal Visitors</strong><br>
+          (3 or More Sessions/User)
+        </th>
+        <th colspan="2">
+          <strong>Fans</strong><br>
+          (4 or More Sessions/User)
+        </th>
+        <th colspan="2">
+          <strong>Superfans</strong><br>
+          (5 or More Sessions/User)
+        </th>
+      </tr>
+      <tr>
+        <th><strong>Website</strong></th>
+        <th><strong>Users</strong></th>
+        <th><strong>Pageviews</strong></th>
+        <th><strong>Users</strong></th>
+        <th><strong>Pageviews</strong></th>
+        <th><strong>Users</strong></th>
+        <th><strong>Pageviews</strong></th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>AméricaEconomía</td>
+        <td>+16.0%</td>
+        <td>+10.4%</td>
+        <td>+11.8%</td>
+        <td>+6.8%</td>
+        <td>+6.9%</td>
+        <td>+4.6%</td>
+      </tr>
+      <tr>
+        <td>EHN</td>
+        <td>+24.2%</td>
+        <td>+13.8%</td>
+        <td>+24.9%</td>
+        <td>+13.4%</td>
+        <td>+24.0%</td>
+        <td>+9.0%</td>
+      </tr>
+      <tr>
+        <td>PAPER Magazine</td>
+        <td>+24.7%</td>
+        <td>+20.0%</td>
+        <td>+39.7%</td>
+        <td>+32.1%</td>
+        <td>+55.7%</td>
+        <td>+35.7%</td>
+      </tr>
+      <tr>
+        <td>Second Nexus</td>
+        <td>+16.8%</td>
+        <td>+14.2%</td>
+        <td>+15.0%</td>
+        <td>+13.4%</td>
+        <td>+15.7%</td>
+        <td>+13.3%</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+## From normal to high-performing: How RebelMouse transforms websites
+
+The good news is that performing well on Core Web Vitals is possible with thoughtful, strategic changes made to your site's codebase. Here are a few ways RebelMouse powers high-performing websites:
+
+- **Progressive web app:** RebelMouse optimizes all third-party calls in a progressive web app that delivers the most important content first, and then sequences the rest of the calls as the reader needs to see them.
```

Comment:
This section is all nested lists some with only one item. It might read more nicely with subheads instead of the nested list structure?